### DISCUSSION TITLE
ID order for BT-26(a)-Procedure/BT-263-Procedure and others

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#157] [Q&A] [answered]
Hi !
it is possible changing id for BT-26(a)-Procedure/BT-263-Procedure, BT-26(a)-Lot/BT-263-Lot , BT-26(m)-Lot/BT-262-Lot ?
I mean, before the xml tag and after the listName attribute, as you did in BT-02-notice/BT-03-notice.

Sorting the sdkFields by id,  it comes before the attribute.
Thanks
Renato

**Comment by rouschr** [comment, marked as answer]
In the SDK xsds have no notion of attribute order, I also do no think that XML attributes have a notion of order at all.
Attributes are key/value (get or set by attribute name), and I have never seen any program rely on the XML attribute order.
For the order of elements, you must rely on the the SDK xsd order (sequences), this will be easier to do once SDK 1.6 comes out:
https://github.com/OP-TED/eforms-notice-editor/blob/develop/src/test/java/eu/europa/ted/eforms/noticeeditor/helper/notice/NoticeXmlTagSorterTest.java (note that the Java package location may change later)
